charts/jd-bureau/values.yaml

```diff
@@ -11,4 +11,13 @@ nodejs:
     SMARTSURVEY_EXPORT_V2: "Dashboard Export"
   ingressSessionAffinity:
     enabled: true
-    sessionCookieName: affinity
\ No newline at end of file
+    sessionCookieName: affinity
+  keyVaults:
+    juror-digital-vault-{{ .Values.global.environment }}
```

Comment:
environment should be appended automatically

Reply:
shall i leave the name as "juror-digital-vault"

Reply:
yeah it's in their code, best to live with that :p